### ISSUE TITLE
Unable to set default timeout value in pulseeffects GUI (4.7.1)

### ISSUE THREAD

**Issue by alexbramford** [wwmm/easyeffects#1123]
pulseeffects appears to be "dropping" the connection when I switch between sources (e.g. Chrome --> Rhythmbox --> Chrome)
or when a source has been inactive longer than 30 seconds.

There seems to be no option to change the PulseEffects --> Settings --> General --> **Activity Timeout** since the setting is **not there** under the "Priority" box

https://imgur.com/a/SGAYDkr

My setup is: Rhythmbox --> pulseaudio (module-raop-sink) --> RaspberryPi (Shairport Sync AirPlay audio player)

pulseeffects: 4.7.1

pulseaudio --version
pulseaudio 14.0-rebootstrapped

Fedora 32

screenfetch -n

 OS: Fedora 
 Kernel: x86_64 Linux 5.11.22-100.fc32.x86_64
 Uptime: 2h 49m
 Packages: 3695
 Shell: bash 5.0.17
 Resolution: 1920x1080
 DE: KDE 5.75.0 / Plasma 5.18.5
 WM: KWin
 WM Theme: Breeze Dark
 GTK Theme: kde-pingguo [GTK3]
 Icon Theme: breeze-dark
 Disk: 649G / 924G (74%)
 CPU: Intel Core i7-7500U @ 4x 3.5GHz [68.0°C]
 GPU: Mesa Intel(R) HD Graphics 620 (KBL GT2)
 RAM: 4876MiB / 31979MiB

**Comment by vchernin** [comment]
This doesn't answer your question, but you are running Fedora 32 which is unsupported since May 2021. I'd really recommend not to do that and upgrade to 33 or 34...

**Comment by wwmm** [comment]
Considering that `4.7.1` is being used the most likely cause of the problem is an upgrade of the package `dconf` to a version equal or higher than `0.40`. In that case you will need PulseEffects `4.8.5` if you are on Pulseaudio. If you are using PipeWire I recommend moving to EasyEffects.

**Comment by alexbramford** [comment]
Yup - F34 is a sore topic for me...

Usually I keep my system bang up to date. But right now I'm only on Fedora 32 because pipewire on F33 and F34 does not currently support the pulseaudio **module-raop-sink** module I'm using to stream to my RaspberryPi. 

https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/1542


cat ~/.pulse/default.pa

load-module module-raop-sink server=192.168.1.109 protocol=UDP codec=ALAC rate=44100 sink_name=moode-audio sink_properties='device    .description="192.168.1.109-moode_audio" tsched=0 fixed_latency_range=no fragments=12 fragment_size=1024

**Comment by vchernin** [comment]
FYI Fedora 33 does not use PipeWire by default.

On 34 which default to PipeWire you can switch to PulseAudio with `sudo dnf swap --allowerasing pipewire-pulseaudio pulseaudio`

And to get PulseEffects 4.7.5 on Fedora 32 you could try the [Flatpak](https://flathub.org/apps/details/com.github.wwmm.pulseeffects).

**Comment by alexbramford** [comment]
@vchernin I'm keen to switch to F34, so `sudo dnf swap --allowerasing pipewire-pulseaudio pulseaudio` is most likely the way forward. 

I presume that will install the original pulseaudio codebase so that I can continue to use `module-raop-sink`?

(This post is veering off-topic now... I'll go the research on the F34 dnf swap before I attempt to destroy anything...)

Thanks so much @vchernin and @wwmm for your valuable help

**Comment by vchernin** [comment]
> I presume that will install the original pulseaudio codebase so that I can continue to use `module-raop-sink`?

Yes should get the original PulseAudio codebase as well as modules like that. I haven't tried that but I would be surprised if it didn't work. Also do make sure you reboot after running that.

**Comment by alexbramford** [comment]
An update...

So, I'm "prototyping" Fedora 34 + pulseaudio + pulseffects in VirtualBox before I commit to upgrading my laptop.

After "switching back" to pulseaudio as above, I find that the **module-raop-sink** is "supported" and I can stream from F34 to moodeAudio 7.3.0 on the RaspberryPi, per my current setup.

I will have to wait until pipewire supports this module, but for now this will suffice.

Next up, pulseeffects on F34...
